### DISCUSSION TITLE
The same skin displays differently on Apple and non-Apple devices

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#30148] [Q&A] [answered]
on iOS(iPadOS) and MacOS:

| on iOS(iPadOS) and MacOS | on Windows |
| -- | -- |
| ![on iOS(iPadOS) and MacOS](https://github.com/user-attachments/assets/e41b524a-7b51-42df-b049-6ba4202bb3c8) | ![on Windows](https://github.com/user-attachments/assets/b0c40cb3-a581-4919-a7e7-f5e4b6a8fdf1) |

The fruit is noticeably darker

| on iOS(iPadOS) and MacOS | on Windows |
| -- | -- |
| ![on iOS(iPadOS) and MacOS](https://github.com/user-attachments/assets/63c27321-da63-44a7-95d9-8ef93d38603d) | ![on Windows](https://github.com/user-attachments/assets/5bf2d972-71ea-4398-a812-884e9030e06c) |

The area around the cursor(and tail) is noticeably darker

std skin: https://files.catbox.moe/01isxd.osk
ctb skin: https://files.catbox.moe/f7172e or https://www.swisstransfer.com/d/d800340d-c3b0-4244-92ca-56a4a3932087

**Comment by peppy** [comment]
This implies that we might still be doing premultiplied alpha incorrectly.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/issues/30399
